fix: Sanitize Java keywords in generated client

rahulsom opened the pull request:
Prior to this change, the generated code would have the right name for the parameter but wrong name in the creation of InputArgument.

```java
InputArgument packageArg = new InputArgument("package", package, false, null);
```

Now we have a test for generated code being compilable, and we handle this case correctly.

sjohnr reviewed (commented):
Hi @rahulsom, thanks for the PR! I added a comment for your review below.

rahulsom committed:
fix: Sanitize Java keywords in generated client

Prior to this change, the generated code would have the right name for the parameter but wrong name in the creation of InputArgument.

```java
InputArgument packageArg = new InputArgument("package", package, false, null);
```

Now we have a test for generated code being compilable, and we handle this case correctly.
